src/util.ts

```diff
@@ -301,49 +301,6 @@ export function getLinkFromParams(
 	return link
 }
 
-/**
- * Returns the deposit index from a tx receipt
- */
-export function getDepositIdx(txReceipt: any, chainId: number | string, contractVersion: string) {
```

Comment:
You removed this, why? 
Be careful with removing entire functions that are exported, this should explicitly be mentioned in the changelog since integrators might be using this.

Reply:
Makes sense, yeah. Restored it and marked as deprecated